wgengine/magicsock/magicsock.go

```diff
@@ -1487,12 +1544,22 @@ func (c *Conn) periodicReSTUN() {
 	}
 	timer := time.NewTimer(dur())
 	defer timer.Stop()
+	var lastIdleState interface{} // nil or bool
```

Comment:
*bool?

Reply:
actually I remembered we had the opt.Bool type, so used that instead.